generator/source/_posts/2018-09-19-exams_coming_soon.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
This post would be a better example since it's Sawtooth-centric:

https://www.hyperledger.org/blog/2018/07/24/hyperledger-sawtooth-seth-and-truffle-101